src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukSummaryList.scala.html

```diff
@@ -14,11 +14,46 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.Card
+
 @this()
 
 @(params: SummaryList)
 @import params._
-<dl class="@toClasses("govuk-summary-list", classes)"@toAttributes(attributes)>
+
+@summaryCard(card: Card) = {
+  <div class="@toClasses("govuk-summary-card", card.classes)"@toAttributes(card.attributes)>
+    <div class="govuk-summary-card__title-wrapper">
+      @if(card.title.isDefined) {
+      <h@{card.title.get.headingLevel.getOrElse(2)} class="@toClasses("govuk-summary-card__title", card.title.get.classes)">
+        @card.title.get.content.asHtml
+      </h@{card.title.get.headingLevel.getOrElse(2)}>
+       }
```

Comment:
would this work?
```suggestion
      @card.title.map { title =>
      <h@{title.headingLevel.getOrElse(2)} class="@toClasses("govuk-summary-card__title", title.classes)">
        @title.content.asHtml
      </h@{title.headingLevel.getOrElse(2)}>
       }
```

Reply:
@ellamdav Perfect, good suggestion, rewritten as per your proposal.